src/__tests__/server.test.ts

```diff
@@ -72,37 +100,10 @@ describe("Server graceful shutdown", () => {
     app.emit("server:stop");
```

Comment:
It seems that, after all tests executions ( one test in this case ), a process is still running after tests finish. I suppose that after emit of server:stop event it does not wait for `server.close()`.